### PR TITLE
Tidy tokens module

### PR DESCRIPTION
**Pull request by adamchainz**
Improve order of functions, some names, and document each.

### CODE REVIEW

**Comment by adamchainz** [comment]
@browniebroke any thoughts?

File: src/django_upgrade/tokens.py [hunk 1]
```diff
@@ -157,13 +199,17 @@ def replace_arguments(
                 raise AssertionError(f"{keyword.arg} argument not found")
 
 
-def update_imports(
+def update_import_names(
```
Comment (browniebroke) [inline src/django_upgrade/tokens.py:202]: This name sounds a lot clearer to me 👍🏻

File: src/django_upgrade/fixers/empty_result_set_exception.py [hunk 2]
```diff
@@ -42,5 +42,5 @@ def visit_ImportFrom(
         and any(alias.name in REWRITES[node.module] for alias in node.names)
     ):
         yield ast_start_offset(node), partial(
-            rewrite_imports, node=node, rewrites=REWRITES
+            update_import_modules, node=node, module_rewrites=REWRITES[node.module]
```
Comment (browniebroke) [inline src/django_upgrade/fixers/empty_result_set_exception.py:45]: Ah yes good point, no need to pass the whole `REWRITES` dict 👍🏻

**Review by browniebroke — COMMENTED** [review]
Nice simplifications, names are a bit clearer.

**Comment by adamchainz** [comment]
Thanks for having a read!

**Pull request merged**